plugins/coverbrowser.koplugin/main.lua

```diff
@@ -368,6 +368,42 @@ function CoverBrowser:addToMainMenu(menu_items)
                     }
                 }
             },
+            {
+                text = _("Consistent aspect ratio of book covers"),
+                checked_func = function()
+                    return BookInfoManager:getSetting("consistent_aspect_ratio_covers")
```

Comment:
I think it needs some other wording. "Consistent" sounds like a good thing to me - and seeing that wording I would probably check it, while here it means that the user will get a consistently screwed up aspect ratio :)

Reply:
Forced?

Reply:
I'd possibly wnt to stack *both* force + consistent, *and* add an `help_text` to confirm that this does indded mean "strech things to make everything look like ass" ;p (/s about the opinionated wording, but I do mean the first bit, e.g., an help_text that mpentions stretching).

Reply:
And of course that it'll tank browsing performance, see main discussion.